bot/cogs/tags_cog.py

```diff
@@ -188,7 +188,8 @@ async def info(self, ctx, name):
 
         tag = await repo.get_tag(name, ctx.guild.id)
 
-        author = self.bot.get_user(tag['fk_UserId'])
+        #uses the fetch_user API call so it can get tag information for tags where the owner left the server
+        author = await self.bot.fetch_user(tag['fk_UserId'])
```

Comment:
This needs to be handled differently, specificlaly it needs to be part of the tag owner improvemnet

Reply:
How so?
My understanding is that the original function call used only worked on members currently in the server because it pulled them from their guild

Reply:
thats correct, but if theyve left the server then we dont want to still reference them. the tag needs to be "unowned" which is the tag claim functionality thats been planned

Reply:
Ah gotcha. I'll revert the change